### DISCUSSION TITLE
【灌水】抽青衣歪了11号，望不周知。

### DISCUSSION THREAD

**Discussion by HKFDBHZN** [share121/inter-knot#2367] [其他]
啊啊啊本来胜券在握，结果...我的青衣啊。
甚至还发现我的朱鸢上海不如别人的安东QAQ
我还是去玩我的崩铁吧。

**Comment by glhongbo** [comment]
我歪了猫又

**Comment by yuling3378** [comment]
我中了

**Comment by 6beihongcha** [comment]
已阅

**Comment by liuying5201314** [comment]
我的青衣是只猫<img src="https://bbs-static.miyoushe.com/static/2024/03/21/6431de7537491cb394a82e8a0ff8cf77_9185062232016213590.png" alt="undefined" width="75" >

**Reply by HKFDBHZN** [reply]
6

**Comment by 13671406204** [comment]
3期up全歪，介绍一下（歪丽娜）

**Comment by 13671406204** [comment]
3期up全歪

**Comment by Rain-YYC** [comment]
已阅，已歪<img src="https://img-static.mihoyo.com/communityweb/upload/ad0c5f768b9022b8b9487c3dfb3c8b59.png" alt="倒" width="75" >

**Comment by Random114** [comment]
已阅 还没抽到

**Comment by Random114** [comment]
已阅 还没抽到

**Comment by wtnl551** [comment]
歪了丽娜，但又把青衣肝出来了

**Comment by hua-yuanshuai** [comment]
从开发到现在只抽过20抽，现在仍然在等雅
